front/app/components/FormBuilder/components/FormBuilderToolbox/ToolboxItem.tsx

```diff
@@ -26,18 +27,19 @@ const AddIcon = styled(Icon).attrs({ name: 'plus' })`
   margin-right: 0;
 `;
 
-const StyledBox = styled(Box)`
+const StyledBox = styled(Box)<{ disabled: boolean }>`
   text-align: left;
   ${AddIcon} {
     visibility: hidden;
   }
   &:hover {
-    background-color: ${colors.grey200};
+    background-color: ${({ disabled }) =>
```

Comment:
Do we want to turn off the hover here?

Reply:
Yes that's the intention (at least visually hiding it) :thinking: Do you have a suggestion for a better way to do that?

Reply:
I think the cursor should be enough. At the moment it looks like a whole block disabled

Reply:
Minor though, maybe we go with it as is and then get feedback